app/components/Modal/DepositModal.jsx

```diff
@@ -138,25 +139,19 @@ class DepositModalContent extends DecimalChecker {
             return;
         }
 
-        if (selectedGateway == "OPEN") {
-            this.setState({
-                isOpenledger: true
-            });
-            let depositAddress = this.deposit_address_cache.getCachedInputAddress(
-                selectedGateway.toUpperCase(),
+        let depositAddress;
+        if(selectedGateway && selectedAsset) {
+            depositAddress = this.deposit_address_cache.getCachedInputAddress(
+                selectedGateway.toLowerCase(),
```

Comment:
Why was this changed to lowercase? You don't have to change it because I just looked at the storage keys and it seems we're being a bit inconsistent with the use of the deposit cache so I'll create another issue for cleaning it up.

Reply:
It's a bit inconsistent, yes. We better take a closer look to make it cleaned up. I think it's lower case in some places but as you point out, it was upper case before.

Reply:
I would suggest we always compare lowercase to lowercase.